interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/rich-text.md

```diff
@@ -46,3 +46,79 @@ The Rich Text also enables you to merge data fields and static content in its te
 ![](../../../../../images/eurd-win-rich-text-mail-merge.png)
 
 See the [Bind Controls to Data](../../bind-to-data/bind-controls-to-data-expression-bindings.md) and [Use Embedded Fields](../../bind-to-data/use-embedded-fields-mail-merge.md) topics for more information.
+
+## Markup Text
+
+### Supported Tags
+
+The table below lists the supported HTML tags. External links are processed for inline pictures and style sheets (CSS files). The ID and Class attributes are interpreted for all tags, including the unlisted ones. These attributes are used to specify a style applied to the document part defined by a certain tag.
```

Comment:
"These attributes are used to specify a style applied to the document part defined by a certain tag." This does not sound right

Reply:
Changed to:
> These attributes are used to specify a style for content within a certain tag